jujugui/static/gui/src/app/components/help/tour/tour.js

```diff
@@ -0,0 +1,156 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const Lightbox = require('../../lightbox/lightbox');
+const SvgIcon = require('../../svg-icon/svg-icon');
+
+class Tour extends React.PureComponent {
+  render() {
+    const staticURL = this.props.staticURL;
+    const basePath = `${staticURL}/static/gui/build/app/assets/images/non-sprites/tour`;
+    return (
+      <div>
+        <span className="back-to-help"
+          onClick={this.props.endTour.bind(this)}>
+          <SvgIcon
+            name="chevron_down_16" size="16" className="back-to-help__icon" />
+          Back to GUI help
+        </span>
+        <Lightbox
+          close={this.props.close.bind(this)}
+          extraClasses={['tour']}>
+          <div className="tour__slide">
+            <img className="tour__slide-image"
+              src={`${basePath}/welcome@1x.png`}
+              srcSet={`${basePath}/welcome@2x.png 2x`}
+            />
+            <div className="tour__slide-description clearfix">
+              <p className="ten-col">
```

Comment:
ten-col is not wide enough for this section.

<img width="885" alt="screen shot 2017-10-24 at 3 06 19 pm" src="https://user-images.githubusercontent.com/532033/31968000-ee710090-b8cc-11e7-98b1-34c7fbbb83c8.png">

Reply:
It was designed at ten-col and passed design review so going to go ahead and land it.